ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -69,6 +70,58 @@ do
   scp "${SSHOPTS[@]}" "${item}" "root@${IP}:manifests/${manifest##manifest_}"
 done <   <( find "${SHARED_DIR}" \( -name "manifest_*.yml" -o -name "manifest_*.yaml" \) -print0)
 
+
+# dev-scripts setup for baremetal clusters
```

Comment:
I'd suggest to move any eventual change strictly related to dev-script setup in a separate PR